core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -43,7 +43,7 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, csscla
             }
 
             f.section(title: _("Agents")) {
-                f.entry(title: _("TCP port for JNLP agents"), field: "slaveAgentPort") {
+                f.entry(title: _("TCP port for inbound agents"), field: "slaveAgentPort") {
```

Comment:
One of the more prominent GUI changes.

Note that this is technically inaccurate since the TCP port _can_ be used for other purposes:

* Remoting-based CLI, since removed.
* CloudBees Jenkins Operations Center.

In practice it seems sufficient to label it according to what is by far its most important use.